src/app/lib/button/button-demo/button-demo.component.html

```diff
@@ -7,17 +7,17 @@ <h5>
     </h5>
     <div class="sample-buttons">
         <button hc-button title="Primary Button" color="primary">Primary</button>
-        <button hc-button title="Primary-Alt1 Button" color="primary-alt1">Primary Alt1</button>
-        <button hc-button title="Primary-Alt2 Button" color="primary-alt2">Primary Alt2</button>
-        <button hc-button title="Primary-Alt3 Button" color="primary-alt3" >Primary Alt3</button>
+        <button hc-button title="Primary-Alt Button" color="primary-alt">Primary Alt</button>
+        <button hc-button title="Destructive Button" color="destructive">Destructive</button>
```

Comment:
I like the descriptive labels for the different primary button colors. It makes it seem less like a whimsical choice and more like the choice should be informed by the current context.